### ISSUE TITLE
several routes

### ISSUE THREAD

**Issue by benjyz** [yamandf/pancakeswap-py#1]
the route array has 3 entries? 
https://github.com/yamandf/pancakeswap-py/blob/main/pancakeswap.py#L181

**Comment by yamandf** [comment]
Yes, my understanding is for this call to swapExactTokensForTokens it's is more likely for there to be an LP for an arbitrary token paired with the base network token (ETH / BNB) versus a pair that's token0<->token1. Although, could augment this by adding a check confirming an LP exists with enough liquidity to complete the swap; if that check fails it could default to the three address path array.
